guides/common/modules/proc_configuring-the-project-client-in-keycloak.adoc

```diff
@@ -0,0 +1,35 @@
+[id="configuring-the-project-client-in-keycloak_{context}"]
+= Configuring the {Project} Client in {Keycloak}
+
+Use this procedure to configure the {Project} client in the {Keycloak} web UI and create group and audience mappers for the {Project} client.
+
+.Procedure
+
+. In the {Keycloak} web UI, navigate to *Clients* and click the {Project} client.
+. Configure access type:
+* If you plan to configure {Project} settings using the web UI, from the *Access Type* list, select *confidential*.
+* If you plan to configure {Project} settings using the CLI, from the *Access Type* list, select *public*.
+. Locate the *Valid redirect URI* field.
+This field must contain one redirect URI by default in the following form: `https://{foreman-example-com}/users/extlogin/redirect_uri`.
```

Comment:
For Hammer CLI we need the redirect URI as `urn:ietf:wg:oauth:2.0:oob`, we need to add this :+1: